src/annotator/sidebar.js

```diff
@@ -18,8 +18,16 @@ import BucketBar from './plugin/bucket-bar';
  * @prop {number} height
  */
 
+/**
```

Comment:
Can you help me understand the alternate `@typdef` syntax here? I find it a little harder to read and it's formatted funky in my editor (VSCode), but perhaps there is a reason to do it like this?

![image](https://user-images.githubusercontent.com/439947/107264526-0f61fb80-6a11-11eb-8c52-b3f586503b72.png)

Can we use our normal conventions if there isn't a reason to do this one differently?

Reply:
No problem, I will change that.